3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py

```diff
@@ -123,15 +123,21 @@
         output_filename=output_annotated_file,
         output_type="parquet",
     )
-    
+
     # Step 2: Normalization
-    normalized_df = normalize(
-        profiles=output_annotated_file,
-        method="standardize",
-        output_file=output_normalized_file,
-        output_type="parquet",
-    )
-    
+    norm_args = {
```

Comment:
Nice change here! But, I think that we should keep the formatting of the code consistent, so I would recommend instead of creating a dictionary, do something like this:

```
# set default for samples to use in normalization
    samples = "all"
     
# Only for Plate 4, we want to normalize to no siRNA treatment (controls)
    if plate == "Plate_4":
        samples = "Metadata_Concentration == 0.0"

    print(f"Performing normalization for", plate, "using this samples parameter:", samples)

# Step 2: Normalization
    normalized_df = normalize(
        profiles=output_annotated_file,
        method="standardize",
        output_file=output_normalized_file,
        output_type="parquet",
        samples=samples,
    )
 ```

Reply:
That works, changed! It looks like the parquet files didn't change, so I can only upload the code files.